fs-diff-test-utils/Dockerfile.podman-image

```diff
@@ -0,0 +1,3 @@
+FROM centos:8.3.2011
+RUN yum -y install podman
+COPY diff-fs.sh /diff-fs.sh
```

Comment:
This would fall under "testing the test suite", which is inherently a CI task.  I don't think we should include this in this commit, as this PR focuses on checking containers for the CNF under test.